master/internal/config/config.go

```diff
@@ -72,6 +74,12 @@ type DBConfig struct {
 	SSLRootCert string `json:"ssl_root_cert"`
 }
 
+// WebhooksConfig hosts configuration fields for webhook functionality.
+type WebhooksConfig struct {
+	BaseURL    string `json:"base_url"`
+	SigningKey string `json:"signing_key"`
```

Comment:
@ioga I began making this update, how do I handle the flags? I had to remove the flags https://github.com/determined-ai/determined/pull/5258/commits/6fd437ea8e435168a7adec6b3f7cc6b6d1912e2c#diff-5d11e06d39592d7df703d9e9ed7d3c5c9210b7e39093128e3fa50383c6aba26cL119

for example how do I correctly set up
```
registerString(flags, name("webhooks", "base-url"),
		defaults.Webhooks.BaseURL, "Base URL to access Determined Web Application")
```
 since there are expecting pointers that could be nil if no value is specified?